mocks/service/launchpad/index.js

```diff
@@ -29,12 +29,18 @@ router.get('/devel/*snaps', (req, res) => {
         {
           resource_type_link: `${base}/#snap`,
           git_repository_url: `${prefix}mock-repo-1`,
-          self_link: `${base}/${owner}/+snap/mock-snap-1`
+          self_link: `${base}/${owner}/+snap/mock-snap-1`,
+          name: 'mock-snap-1'
         },
         {
           resource_type_link: `${base}/#snap`,
           git_repository_url: `${prefix}mock-repo-2`,
-          self_link: `${base}/${owner}/+snap/mock-snap-1`
+          self_link: `${base}/${owner}/+snap/mock-snap-2`,
+          name: 'mock-snap-2'
+        },
+        {
+          resource_type_link: `${base}/#snap`,
+          git_repository_url: `${prefix}mock-repo-3`
```

Comment:
Build status can be a plain link to repo's builds page for now.

I guess it would be good to have placeholders for the others too already. We need them anyway, and we need them soon.